system-config-printer fails to build in nixos-unstable

**Describe the bug**
[system-config-printer](https://github.com/NixOS/nixpkgs/blob/master/pkgs/tools/misc/system-config-printer/default.nix) fails to build in the latest nixos-unstable (https://github.com/NixOS/nixpkgs/commit/16105403bdd843540cbef9c63fc0f16c1c6eaa70). This seems to be caused by a failing test:
```
system-config-printer> FAIL: test-xml-validity.sh
system-config-printer> ==========================
system-config-printer> ./xml/preferreddrivers.xml validates
system-config-printer> Validating ./xml/preferreddrivers.xml
system-config-printer> Traceback (most recent call last):
system-config-printer>   File "/build/source/./xml/validate.py", line 79, in <module>
system-config-printer>     if not validator.validate ():
system-config-printer>   File "/build/source/./xml/validate.py", line 38, in validate
system-config-printer>     (drivertypes, preferenceorder) = preferreddrivers.getchildren ()
system-config-printer> AttributeError: 'xml.etree.ElementTree.Element' object has no attribute 'getchildren'
system-config-printer> FAIL test-xml-validity.sh (exit status: 1)
```

I'm running `git bisect` right now and will update this issue when it's done.

**To Reproduce**
Steps to reproduce the behavior:
1. `nix build github:NixOS/nixpkgs/16105403bdd843540cbef9c63fc0f16c1c6eaa70#system-config-printer`

**Additional context**
<details>
 <summary>Complete build logs</summary>

```
~ λ nix build github:NixOS/nixpkgs/16105403bdd843540cbef9c63fc0f16c1c6eaa70#system-config-printer --print-build-logs
system-config-printer> unpacking sources
system-config-printer> unpacking source archive /nix/store/2ymc2dl01hf475n9n183w2c9aix1wrij-source
system-config-printer> source root is source
system-config-printer> patching sources
system-config-printer> applying patch /nix/store/37p56pi9i0n8xg7xnrn43dqyqq9w5k0r-detect_serverbindir.patch
system-config-printer> patching file cupshelpers/config.py.in
system-config-printer> applying patch /nix/store/3w75lp2fldbfkk5iq6429351ip5sfqxg-cf9903466c1a2d18a701f3b5e8c7e03483e1244d.patch
system-config-printer> patching file NEWS
system-config-printer> patching file udev/udev-configure-printer.c
system-config-printer> applying patch /nix/store/937fznmf2mnqg03lsrmnc44id0ykjq43-b9289dfe105bdb502f183f0afe7a115ecae5f2af.patch
system-config-printer> patching file udev/udev-configure-printer.c
system-config-printer> autoreconfPhase
system-config-printer> autoreconf: Entering directory `.'
system-config-printer> autoreconf: configure.ac: not using Gettext
system-config-printer> autoreconf: running: aclocal --force
system-config-printer> autoreconf: configure.ac: tracing
system-config-printer> autoreconf: configure.ac: AM_GNU_GETTEXT is used, but not AM_GNU_GETTEXT_VERSION
system-config-printer> autoreconf: configure.ac: not using Libtool
system-config-printer> autoreconf: running: /nix/store/0450sqn6d5gijifbxn6w9jy2qiqg8wwk-autoconf-2.69/bin/autoconf --force
system-config-printer> autoreconf: configure.ac: not using Autoheader
system-config-printer> autoreconf: running: automake --add-missing --copy --force-missing
system-config-printer> configure.ac:5: installing './compile'
system-config-printer> configure.ac:3: installing './missing'
system-config-printer> Makefile.am:397: warning: PYTEST-py.test: non-POSIX variable name
system-config-printer> Makefile.am: installing './INSTALL'
system-config-printer> Makefile.am: installing './depcomp'
system-config-printer> parallel-tests: installing './test-driver'
system-config-printer> autoreconf: Leaving directory `.'
system-config-printer> configuring
system-config-printer> configure flags: --disable-dependency-tracking --prefix=/nix/store/76v2wq6hlyi4za9jhhj4sf9m630f76g4-system-config-printer-1.5.12 --with-udev-rules --with-udevdir=/nix/store/76v2wq6hlyi4za9jhhj4sf9m630f76g4-system-config-printer-1.5.12/etc/udev --with-systemdsystemunitdir=/nix/store/76v2wq6hlyi4za9jhhj4sf9m630f76g4-system-config-printer-1.5.12/etc/systemd/system
system-config-printer> checking for a BSD-compatible install... /nix/store/nvlnl2swxmk50x25mzn71rhc0gzpylfr-coreutils-8.32/bin/install -c
system-config-printer> checking whether build environment is sane... yes
system-config-printer> checking for a thread-safe mkdir -p... /nix/store/nvlnl2swxmk50x25mzn71rhc0gzpylfr-coreutils-8.32/bin/mkdir -p
system-config-printer> checking for gawk... gawk
system-config-printer> checking whether make sets $(MAKE)... yes
system-config-printer> checking whether make supports nested variables... yes
system-config-printer> checking whether NLS is requested... yes
system-config-printer> checking for intltool-update... /nix/store/cf74dzz61z7jahr8qh4hg6yl3swxl9is-intltool-0.51.0/bin/intltool-update
system-config-printer> checking for intltool-merge... /nix/store/cf74dzz61z7jahr8qh4hg6yl3swxl9is-intltool-0.51.0/bin/intltool-merge
system-config-printer> checking for intltool-extract... /nix/store/cf74dzz61z7jahr8qh4hg6yl3swxl9is-intltool-0.51.0/bin/intltool-extract
system-config-printer> checking for xgettext... /nix/store/vyy5f2lnkq1b3r7rr79x1hwwcj2di3w6-gettext-0.21/bin/xgettext
system-config-printer> checking for msgmerge... /nix/store/vyy5f2lnkq1b3r7rr79x1hwwcj2di3w6-gettext-0.21/bin/msgmerge
system-config-printer> checking for msgfmt... /nix/store/vyy5f2lnkq1b3r7rr79x1hwwcj2di3w6-gettext-0.21/bin/msgfmt
system-config-printer> checking for gmsgfmt... /nix/store/vyy5f2lnkq1b3r7rr79x1hwwcj2di3w6-gettext-0.21/bin/msgfmt
system-config-printer> checking for perl... /nix/store/y29hhfh2gjmiagdanzywabjbd2y4g79c-perl-5.34.0/bin/perl
system-config-printer> checking for perl >= 5.8.1... 5.34.0
system-config-printer> checking for XML::Parser... ok
system-config-printer> checking for a sed that does not truncate output... /nix/store/b3l5zq2nkhfpqrr0lil06dhqyjf5jz38-gnused-4.8/bin/sed
system-config-printer> checking for msgfmt... (cached) /nix/store/vyy5f2lnkq1b3r7rr79x1hwwcj2di3w6-gettext-0.21/bin/msgfmt
system-config-printer> checking for gmsgfmt... (cached) /nix/store/vyy5f2lnkq1b3r7rr79x1hwwcj2di3w6-gettext-0.21/bin/msgfmt
system-config-printer> checking for xgettext... (cached) /nix/store/vyy5f2lnkq1b3r7rr79x1hwwcj2di3w6-gettext-0.21/bin/xgettext
system-config-printer> checking for msgmerge... (cached) /nix/store/vyy5f2lnkq1b3r7rr79x1hwwcj2di3w6-gettext-0.21/bin/msgmerge
system-config-printer> checking whether make supports the include directive... yes (GNU style)
system-config-printer> checking for gcc... gcc
system-config-printer> checking whether the C compiler works... yes
system-config-printer> checking for C compiler default output file name... a.out
system-config-printer> checking for suffix of executables...
system-config-printer> checking whether we are cross compiling... no
system-config-printer> checking for suffix of object files... o
system-config-printer> checking whether we are using the GNU C compiler... yes
system-config-printer> checking whether gcc accepts -g... yes
system-config-printer> checking for gcc option to accept ISO C89... none needed
system-config-printer> checking whether gcc understands -c and -o together... yes
system-config-printer> checking dependency style of gcc... none
system-config-printer> checking build system type... x86_64-pc-linux-gnu
system-config-printer> checking host system type... x86_64-pc-linux-gnu
system-config-printer> checking for ld... ld
system-config-printer> checking if the linker (ld) is GNU ld... yes
system-config-printer> checking for shared library run path origin... done
system-config-printer> checking 32-bit host C ABI... no
system-config-printer> checking how to run the C preprocessor... gcc -E
system-config-printer> checking for grep that handles long lines and -e... /nix/store/7lkzid5yncibabddzaw4sj13n342jz1s-gnugrep-3.6/bin/grep
system-config-printer> checking for egrep... /nix/store/7lkzid5yncibabddzaw4sj13n342jz1s-gnugrep-3.6/bin/grep -E
system-config-printer> checking for ELF binary format... yes
system-config-printer> checking for the common suffixes of directories in the library search path... lib,lib,lib64
system-config-printer> checking for CFPreferencesCopyAppValue... no
system-config-printer> checking for CFLocaleCopyPreferredLanguages... no
system-config-printer> checking for GNU gettext in libc... yes
system-config-printer> checking whether to use NLS... yes
system-config-printer> checking where the gettext function comes from... libc
system-config-printer> checking for a Python interpreter with version >= 3... python
system-config-printer> checking for python... /nix/store/yqjhkdarf7p74wcha7aypzsb751n7wl2-python3-3.9.5/bin/python
system-config-printer> checking for python version... 3.9
system-config-printer> checking for python platform... linux
system-config-printer> checking for python script directory... ${prefix}/lib/python3.9/site-packages
system-config-printer> checking for python extension module directory... ${exec_prefix}/lib/python3.9/site-packages
system-config-printer> checking pkg-config is at least version 0.9.0... yes
system-config-printer> checking for glib-2.0... yes
system-config-printer> checking for libudev >= 172... yes
system-config-printer> checking for libusb-1.0... yes
system-config-printer> checking pkg-config is at least version 0.9.0... yes
system-config-printer> checking that generated files are newer than configure... done
system-config-printer> configure: creating ./config.status
system-config-printer> config.status: creating Makefile
system-config-printer> config.status: creating po/Makefile.in
system-config-printer> config.status: creating system-config-printer
system-config-printer> config.status: creating system-config-printer-applet
system-config-printer> config.status: creating install-printerdriver
system-config-printer> config.status: creating dbus/scp-dbus-service
system-config-printer> config.status: creating udev/configure-printer@.service
system-config-printer> config.status: executing po-directories commands
system-config-printer> config.status: creating po/POTFILES
system-config-printer> config.status: creating po/Makefile
system-config-printer> config.status: executing depfiles commands
system-config-printer> config.status: executing po/stamp-it commands
system-config-printer> building
system-config-printer> build flags: SHELL=/nix/store/b45zavallnsvqwjs9wg9xw167jcs0935-bash-4.4-p23/bin/bash
system-config-printer> Making all in po
system-config-printer> make[1]: Entering directory '/build/source/po'
system-config-printer> file=`echo ar | sed 's,.*/,,'`.gmo \
system-config-printer>   && rm -f $file && /nix/store/vyy5f2lnkq1b3r7rr79x1hwwcj2di3w6-gettext-0.21/bin/msgfmt -o $file ar.po
system-config-printer> file=`echo as | sed 's,.*/,,'`.gmo \
system-config-printer>   && rm -f $file && /nix/store/vyy5f2lnkq1b3r7rr79x1hwwcj2di3w6-gettext-0.21/bin/msgfmt -o $file as.po
system-config-printer> file=`echo bg | sed 's,.*/,,'`.gmo \
system-config-printer>   && rm -f $file && /nix/store/vyy5f2lnkq1b3r7rr79x1hwwcj2di3w6-gettext-0.21/bin/msgfmt -o $file bg.po
system-config-printer> file=`echo bn_IN | sed 's,.*/,,'`.gmo \
system-config-printer>   && rm -f $file && /nix/store/vyy5f2lnkq1b3r7rr79x1hwwcj2di3w6-gettext-0.21/bin/msgfmt -o $file bn_IN.po
system-config-printer> file=`echo bn | sed 's,.*/,,'`.gmo \
system-config-printer>   && rm -f $file && /nix/store/vyy5f2lnkq1b3r7rr79x1hwwcj2di3w6-gettext-0.21/bin/msgfmt -o $file bn.po
system-config-printer> file=`echo br | sed 's,.*/,,'`.gmo \
system-config-printer>   && rm -f $file && /nix/store/vyy5f2lnkq1b3r7rr79x1hwwcj2di3w6-gettext-0.21/bin/msgfmt -o $file br.po
system-config-printer> file=`echo bs | sed 's,.*/,,'`.gmo \
system-config-printer>   && rm -f $file && /nix/store/vyy5f2lnkq1b3r7rr79x1hwwcj2di3w6-gettext-0.21/bin/msgfmt -o $file bs.po
system-config-printer> file=`echo ca | sed 's,.*/,,'`.gmo \
system-config-printer>   && rm -f $file && /nix/store/vyy5f2lnkq1b3r7rr79x1hwwcj2di3w6-gettext-0.21/bin/msgfmt -o $file ca.po
system-config-printer> file=`echo cs | sed 's,.*/,,'`.gmo \
system-config-printer>   && rm -f $file && /nix/store/vyy5f2lnkq1b3r7rr79x1hwwcj2di3w6-gettext-0.21/bin/msgfmt -o $file cs.po
system-config-printer> file=`echo cy | sed 's,.*/,,'`.gmo \
system-config-printer>   && rm -f $file && /nix/store/vyy5f2lnkq1b3r7rr79x1hwwcj2di3w6-gettext-0.21/bin/msgfmt -o $file cy.po
system-config-printer> file=`echo da | sed 's,.*/,,'`.gmo \
system-config-printer>   && rm -f $file && /nix/store/vyy5f2lnkq1b3r7rr79x1hwwcj2di3w6-gettext-0.21/bin/msgfmt -o $file da.po
system-config-printer> file=`echo de | sed 's,.*/,,'`.gmo \
system-config-printer>   && rm -f $file && /nix/store/vyy5f2lnkq1b3r7rr79x1hwwcj2di3w6-gettext-0.21/bin/msgfmt -o $file de.po
system-config-printer> file=`echo el | sed 's,.*/,,'`.gmo \
system-config-printer>   && rm -f $file && /nix/store/vyy5f2lnkq1b3r7rr79x1hwwcj2di3w6-gettext-0.21/bin/msgfmt -o $file el.po
system-config-printer> file=`echo en_GB | sed 's,.*/,,'`.gmo \
system-config-printer>   && rm -f $file && /nix/store/vyy5f2lnkq1b3r7rr79x1hwwcj2di3w6-gettext-0.21/bin/msgfmt -o $file en_GB.po
system-config-printer> file=`echo es | sed 's,.*/,,'`.gmo \
system-config-printer>   && rm -f $file && /nix/store/vyy5f2lnkq1b3r7rr79x1hwwcj2di3w6-gettext-0.21/bin/msgfmt -o $file es.po
system-config-printer> file=`echo et | sed 's,.*/,,'`.gmo \
system-config-printer>   && rm -f $file && /nix/store/vyy5f2lnkq1b3r7rr79x1hwwcj2di3w6-gettext-0.21/bin/msgfmt -o $file et.po
system-config-printer> file=`echo fa | sed 's,.*/,,'`.gmo \
system-config-printer>   && rm -f $file && /nix/store/vyy5f2lnkq1b3r7rr79x1hwwcj2di3w6-gettext-0.21/bin/msgfmt -o $file fa.po
system-config-printer> file=`echo fi | sed 's,.*/,,'`.gmo \
system-config-printer>   && rm -f $file && /nix/store/vyy5f2lnkq1b3r7rr79x1hwwcj2di3w6-gettext-0.21/bin/msgfmt -o $file fi.po
system-config-printer> file=`echo fr | sed 's,.*/,,'`.gmo \
system-config-printer>   && rm -f $file && /nix/store/vyy5f2lnkq1b3r7rr79x1hwwcj2di3w6-gettext-0.21/bin/msgfmt -o $file fr.po
system-config-printer> file=`echo gu | sed 's,.*/,,'`.gmo \
system-config-printer>   && rm -f $file && /nix/store/vyy5f2lnkq1b3r7rr79x1hwwcj2di3w6-gettext-0.21/bin/msgfmt -o $file gu.po
system-config-printer> file=`echo he | sed 's,.*/,,'`.gmo \
system-config-printer>   && rm -f $file && /nix/store/vyy5f2lnkq1b3r7rr79x1hwwcj2di3w6-gettext-0.21/bin/msgfmt -o $file he.po
system-config-printer> file=`echo hi | sed 's,.*/,,'`.gmo \
system-config-printer>   && rm -f $file && /nix/store/vyy5f2lnkq1b3r7rr79x1hwwcj2di3w6-gettext-0.21/bin/msgfmt -o $file hi.po
system-config-printer> file=`echo hr | sed 's,.*/,,'`.gmo \
system-config-printer>   && rm -f $file && /nix/store/vyy5f2lnkq1b3r7rr79x1hwwcj2di3w6-gettext-0.21/bin/msgfmt -o $file hr.po
system-config-printer> file=`echo hu | sed 's,.*/,,'`.gmo \
system-config-printer>   && rm -f $file && /nix/store/vyy5f2lnkq1b3r7rr79x1hwwcj2di3w6-gettext-0.21/bin/msgfmt -o $file hu.po
system-config-printer> file=`echo id | sed 's,.*/,,'`.gmo \
system-config-printer>   && rm -f $file && /nix/store/vyy5f2lnkq1b3r7rr79x1hwwcj2di3w6-gettext-0.21/bin/msgfmt -o $file id.po
system-config-printer> file=`echo is | sed 's,.*/,,'`.gmo \
system-config-printer>   && rm -f $file && /nix/store/vyy5f2lnkq1b3r7rr79x1hwwcj2di3w6-gettext-0.21/bin/msgfmt -o $file is.po
system-config-printer> file=`echo it | sed 's,.*/,,'`.gmo \
system-config-printer>   && rm -f $file && /nix/store/vyy5f2lnkq1b3r7rr79x1hwwcj2di3w6-gettext-0.21/bin/msgfmt -o $file it.po
system-config-printer> file=`echo ja | sed 's,.*/,,'`.gmo \
system-config-printer>   && rm -f $file && /nix/store/vyy5f2lnkq1b3r7rr79x1hwwcj2di3w6-gettext-0.21/bin/msgfmt -o $file ja.po
system-config-printer> file=`echo kn | sed 's,.*/,,'`.gmo \
system-config-printer>   && rm -f $file && /nix/store/vyy5f2lnkq1b3r7rr79x1hwwcj2di3w6-gettext-0.21/bin/msgfmt -o $file kn.po
system-config-printer> file=`echo ko | sed 's,.*/,,'`.gmo \
system-config-printer>   && rm -f $file && /nix/store/vyy5f2lnkq1b3r7rr79x1hwwcj2di3w6-gettext-0.21/bin/msgfmt -o $file ko.po
system-config-printer> file=`echo lt | sed 's,.*/,,'`.gmo \
system-config-printer>   && rm -f $file && /nix/store/vyy5f2lnkq1b3r7rr79x1hwwcj2di3w6-gettext-0.21/bin/msgfmt -o $file lt.po
system-config-printer> file=`echo lv | sed 's,.*/,,'`.gmo \
system-config-printer>   && rm -f $file && /nix/store/vyy5f2lnkq1b3r7rr79x1hwwcj2di3w6-gettext-0.21/bin/msgfmt -o $file lv.po
system-config-printer> file=`echo mai | sed 's,.*/,,'`.gmo \
system-config-printer>   && rm -f $file && /nix/store/vyy5f2lnkq1b3r7rr79x1hwwcj2di3w6-gettext-0.21/bin/msgfmt -o $file mai.po
system-config-printer> file=`echo ml | sed 's,.*/,,'`.gmo \
system-config-printer>   && rm -f $file && /nix/store/vyy5f2lnkq1b3r7rr79x1hwwcj2di3w6-gettext-0.21/bin/msgfmt -o $file ml.po
system-config-printer> file=`echo mr | sed 's,.*/,,'`.gmo \
system-config-printer>   && rm -f $file && /nix/store/vyy5f2lnkq1b3r7rr79x1hwwcj2di3w6-gettext-0.21/bin/msgfmt -o $file mr.po
system-config-printer> file=`echo ms | sed 's,.*/,,'`.gmo \
system-config-printer>   && rm -f $file && /nix/store/vyy5f2lnkq1b3r7rr79x1hwwcj2di3w6-gettext-0.21/bin/msgfmt -o $file ms.po
system-config-printer> file=`echo nb | sed 's,.*/,,'`.gmo \
system-config-printer>   && rm -f $file && /nix/store/vyy5f2lnkq1b3r7rr79x1hwwcj2di3w6-gettext-0.21/bin/msgfmt -o $file nb.po
system-config-printer> file=`echo nds | sed 's,.*/,,'`.gmo \
system-config-printer>   && rm -f $file && /nix/store/vyy5f2lnkq1b3r7rr79x1hwwcj2di3w6-gettext-0.21/bin/msgfmt -o $file nds.po
system-config-printer> file=`echo nl | sed 's,.*/,,'`.gmo \
system-config-printer>   && rm -f $file && /nix/store/vyy5f2lnkq1b3r7rr79x1hwwcj2di3w6-gettext-0.21/bin/msgfmt -o $file nl.po
system-config-printer> file=`echo nn | sed 's,.*/,,'`.gmo \
system-config-printer>   && rm -f $file && /nix/store/vyy5f2lnkq1b3r7rr79x1hwwcj2di3w6-gettext-0.21/bin/msgfmt -o $file nn.po
system-config-printer> file=`echo or | sed 's,.*/,,'`.gmo \
system-config-printer>   && rm -f $file && /nix/store/vyy5f2lnkq1b3r7rr79x1hwwcj2di3w6-gettext-0.21/bin/msgfmt -o $file or.po
system-config-printer> file=`echo pa | sed 's,.*/,,'`.gmo \
system-config-printer>   && rm -f $file && /nix/store/vyy5f2lnkq1b3r7rr79x1hwwcj2di3w6-gettext-0.21/bin/msgfmt -o $file pa.po
system-config-printer> file=`echo pl | sed 's,.*/,,'`.gmo \
system-config-printer>   && rm -f $file && /nix/store/vyy5f2lnkq1b3r7rr79x1hwwcj2di3w6-gettext-0.21/bin/msgfmt -o $file pl.po
system-config-printer> file=`echo pt_BR | sed 's,.*/,,'`.gmo \
system-config-printer>   && rm -f $file && /nix/store/vyy5f2lnkq1b3r7rr79x1hwwcj2di3w6-gettext-0.21/bin/msgfmt -o $file pt_BR.po
system-config-printer> file=`echo pt | sed 's,.*/,,'`.gmo \
system-config-printer>   && rm -f $file && /nix/store/vyy5f2lnkq1b3r7rr79x1hwwcj2di3w6-gettext-0.21/bin/msgfmt -o $file pt.po
system-config-printer> file=`echo ro | sed 's,.*/,,'`.gmo \
system-config-printer>   && rm -f $file && /nix/store/vyy5f2lnkq1b3r7rr79x1hwwcj2di3w6-gettext-0.21/bin/msgfmt -o $file ro.po
system-config-printer> file=`echo ru | sed 's,.*/,,'`.gmo \
system-config-printer>   && rm -f $file && /nix/store/vyy5f2lnkq1b3r7rr79x1hwwcj2di3w6-gettext-0.21/bin/msgfmt -o $file ru.po
system-config-printer> file=`echo si | sed 's,.*/,,'`.gmo \
system-config-printer>   && rm -f $file && /nix/store/vyy5f2lnkq1b3r7rr79x1hwwcj2di3w6-gettext-0.21/bin/msgfmt -o $file si.po
system-config-printer> file=`echo sk | sed 's,.*/,,'`.gmo \
system-config-printer>   && rm -f $file && /nix/store/vyy5f2lnkq1b3r7rr79x1hwwcj2di3w6-gettext-0.21/bin/msgfmt -o $file sk.po
system-config-printer> file=`echo sl | sed 's,.*/,,'`.gmo \
system-config-printer>   && rm -f $file && /nix/store/vyy5f2lnkq1b3r7rr79x1hwwcj2di3w6-gettext-0.21/bin/msgfmt -o $file sl.po
system-config-printer> file=`echo sr@latin | sed 's,.*/,,'`.gmo \
system-config-printer>   && rm -f $file && /nix/store/vyy5f2lnkq1b3r7rr79x1hwwcj2di3w6-gettext-0.21/bin/msgfmt -o $file sr@latin.po
system-config-printer> file=`echo sr | sed 's,.*/,,'`.gmo \
system-config-printer>   && rm -f $file && /nix/store/vyy5f2lnkq1b3r7rr79x1hwwcj2di3w6-gettext-0.21/bin/msgfmt -o $file sr.po
system-config-printer> file=`echo sv | sed 's,.*/,,'`.gmo \
system-config-printer>   && rm -f $file && /nix/store/vyy5f2lnkq1b3r7rr79x1hwwcj2di3w6-gettext-0.21/bin/msgfmt -o $file sv.po
system-config-printer> file=`echo ta | sed 's,.*/,,'`.gmo \
system-config-printer>   && rm -f $file && /nix/store/vyy5f2lnkq1b3r7rr79x1hwwcj2di3w6-gettext-0.21/bin/msgfmt -o $file ta.po
system-config-printer> file=`echo te | sed 's,.*/,,'`.gmo \
system-config-printer>   && rm -f $file && /nix/store/vyy5f2lnkq1b3r7rr79x1hwwcj2di3w6-gettext-0.21/bin/msgfmt -o $file te.po
system-config-printer> file=`echo th | sed 's,.*/,,'`.gmo \
system-config-printer>   && rm -f $file && /nix/store/vyy5f2lnkq1b3r7rr79x1hwwcj2di3w6-gettext-0.21/bin/msgfmt -o $file th.po
system-config-printer> file=`echo tr | sed 's,.*/,,'`.gmo \
system-config-printer>   && rm -f $file && /nix/store/vyy5f2lnkq1b3r7rr79x1hwwcj2di3w6-gettext-0.21/bin/msgfmt -o $file tr.po
system-config-printer> file=`echo uk | sed 's,.*/,,'`.gmo \
system-config-printer>   && rm -f $file && /nix/store/vyy5f2lnkq1b3r7rr79x1hwwcj2di3w6-gettext-0.21/bin/msgfmt -o $file uk.po
system-config-printer> file=`echo vi | sed 's,.*/,,'`.gmo \
system-config-printer>   && rm -f $file && /nix/store/vyy5f2lnkq1b3r7rr79x1hwwcj2di3w6-gettext-0.21/bin/msgfmt -o $file vi.po
system-config-printer> file=`echo zh_CN | sed 's,.*/,,'`.gmo \
system-config-printer>   && rm -f $file && /nix/store/vyy5f2lnkq1b3r7rr79x1hwwcj2di3w6-gettext-0.21/bin/msgfmt -o $file zh_CN.po
system-config-printer> file=`echo zh_TW | sed 's,.*/,,'`.gmo \
system-config-printer>   && rm -f $file && /nix/store/vyy5f2lnkq1b3r7rr79x1hwwcj2di3w6-gettext-0.21/bin/msgfmt -o $file zh_TW.po
system-config-printer> make[1]: Leaving directory '/build/source/po'
system-config-printer> make[1]: Entering directory '/build/source'
system-config-printer> gcc -DPACKAGE_NAME=\"system-config-printer\" -DPACKAGE_TARNAME=\"system-config-printer\" -DPACKAGE_VERSION=\"1.5.12\" -DPACKAGE_STRING=\"system-config-printer\ 1.5.12\" -DPACKAGE_BUGREPORT=\"\" -DPACKAGE_URL=\"\" -DPACKAGE=\"system-config-printer\" -DVERSION=\"1.5.12\" -DENABLE_NLS=1 -DHAVE_GETTEXT=1 -DHAVE_DCGETTEXT=1 -I.     -I/nix/store/8h40slssggzpvb5lr7irp8xa42qajv88-libusb-1.0.24-dev/include/libusb-1.0 -I/nix/store/6l8vy0rpikcw0j1xw67qf983sqpxcfzp-glib-2.68.3-dev/include/glib-2.0 -I/nix/store/i6z36ml0nm0vl93q9jn40bwjbi5b6r8g-glib-2.68.3/lib/glib-2.0/include -g -O2 -c -o udev/udev_configure_printer-udev-configure-printer.o `test -f 'udev/udev-configure-printer.c' || echo './'`udev/udev-configure-printer.c
system-config-printer> udev/udev-configure-printer.c: In function 'find_matching_device_uris':
system-config-printer> udev/udev-configure-printer.c:1024:3: warning: 'httpConnectEncrypt' is deprecated: Use httpConnect2 instead. [8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wdeprecated-declarations-Wdeprecated-declarations8;;]
system-config-printer>  1024 |   cups = httpConnectEncrypt (cupsServer (), ippPort(), cupsEncryption ());
system-config-printer>       |   ^~~~
system-config-printer> In file included from /nix/store/af9nj6nd93m4zq81kg1nxa5sqxfnkrrd-cups-2.3.3-dev/include/cups/ipp.h:18,
system-config-printer>                  from /nix/store/af9nj6nd93m4zq81kg1nxa5sqxfnkrrd-cups-2.3.3-dev/include/cups/cups.h:27,
system-config-printer>                  from udev/udev-configure-printer.c:36:
system-config-printer> /nix/store/af9nj6nd93m4zq81kg1nxa5sqxfnkrrd-cups-2.3.3-dev/include/cups/http.h:458:17: note: declared here
system-config-printer>   458 | extern http_t  *httpConnectEncrypt(const char *host, int port, http_encryption_t encryption) _CUPS_DEPRECATED_1_7_MSG("Use httpConnect2 instead.");
system-config-printer>       |                 ^~~~~~~~~~~~~~~~~~
system-config-printer> udev/udev-configure-printer.c: In function 'for_each_matching_queue':
system-config-printer> udev/udev-configure-printer.c:1356:3: warning: 'httpConnectEncrypt' is deprecated: Use httpConnect2 instead. [8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wdeprecated-declarations-Wdeprecated-declarations8;;]
system-config-printer>  1356 |   http_t *cups = httpConnectEncrypt (cupsServer (), ippPort (),
system-config-printer>       |   ^~~~~~
system-config-printer> In file included from /nix/store/af9nj6nd93m4zq81kg1nxa5sqxfnkrrd-cups-2.3.3-dev/include/cups/ipp.h:18,
system-config-printer>                  from /nix/store/af9nj6nd93m4zq81kg1nxa5sqxfnkrrd-cups-2.3.3-dev/include/cups/cups.h:27,
system-config-printer>                  from udev/udev-configure-printer.c:36:
system-config-printer> /nix/store/af9nj6nd93m4zq81kg1nxa5sqxfnkrrd-cups-2.3.3-dev/include/cups/http.h:458:17: note: declared here
system-config-printer>   458 | extern http_t  *httpConnectEncrypt(const char *host, int port, http_encryption_t encryption) _CUPS_DEPRECATED_1_7_MSG("Use httpConnect2 instead.");
system-config-printer>       |                 ^~~~~~~~~~~~~~~~~~
system-config-printer> udev/udev-configure-printer.c: In function 'enable_queue':
system-config-printer> udev/udev-configure-printer.c:1542:3: warning: 'httpConnectEncrypt' is deprecated: Use httpConnect2 instead. [8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wdeprecated-declarations-Wdeprecated-declarations8;;]
system-config-printer>  1542 |   http_t *cups = httpConnectEncrypt (cupsServer (), ippPort (),
system-config-printer>       |   ^~~~~~
system-config-printer> In file included from /nix/store/af9nj6nd93m4zq81kg1nxa5sqxfnkrrd-cups-2.3.3-dev/include/cups/ipp.h:18,
system-config-printer>                  from /nix/store/af9nj6nd93m4zq81kg1nxa5sqxfnkrrd-cups-2.3.3-dev/include/cups/cups.h:27,
system-config-printer>                  from udev/udev-configure-printer.c:36:
system-config-printer> /nix/store/af9nj6nd93m4zq81kg1nxa5sqxfnkrrd-cups-2.3.3-dev/include/cups/http.h:458:17: note: declared here
system-config-printer>   458 | extern http_t  *httpConnectEncrypt(const char *host, int port, http_encryption_t encryption) _CUPS_DEPRECATED_1_7_MSG("Use httpConnect2 instead.");
system-config-printer>       |                 ^~~~~~~~~~~~~~~~~~
system-config-printer> udev/udev-configure-printer.c: In function 'remove_queue':
system-config-printer> udev/udev-configure-printer.c:1733:3: warning: 'httpConnectEncrypt' is deprecated: Use httpConnect2 instead. [8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wdeprecated-declarations-Wdeprecated-declarations8;;]
system-config-printer>  1733 |   http_t *cups = httpConnectEncrypt (cupsServer (), ippPort (),
system-config-printer>       |   ^~~~~~
system-config-printer> In file included from /nix/store/af9nj6nd93m4zq81kg1nxa5sqxfnkrrd-cups-2.3.3-dev/include/cups/ipp.h:18,
system-config-printer>                  from /nix/store/af9nj6nd93m4zq81kg1nxa5sqxfnkrrd-cups-2.3.3-dev/include/cups/cups.h:27,
system-config-printer>                  from udev/udev-configure-printer.c:36:
system-config-printer> /nix/store/af9nj6nd93m4zq81kg1nxa5sqxfnkrrd-cups-2.3.3-dev/include/cups/http.h:458:17: note: declared here
system-config-printer>   458 | extern http_t  *httpConnectEncrypt(const char *host, int port, http_encryption_t encryption) _CUPS_DEPRECATED_1_7_MSG("Use httpConnect2 instead.");
system-config-printer>       |                 ^~~~~~~~~~~~~~~~~~
system-config-printer> udev/udev-configure-printer.c: In function 'disable_queue':
system-config-printer> udev/udev-configure-printer.c:1765:3: warning: 'httpConnectEncrypt' is deprecated: Use httpConnect2 instead. [8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wdeprecated-declarations-Wdeprecated-declarations8;;]
system-config-printer>  1765 |   http_t *cups = httpConnectEncrypt (cupsServer (), ippPort (),
system-config-printer>       |   ^~~~~~
system-config-printer> In file included from /nix/store/af9nj6nd93m4zq81kg1nxa5sqxfnkrrd-cups-2.3.3-dev/include/cups/ipp.h:18,
system-config-printer>                  from /nix/store/af9nj6nd93m4zq81kg1nxa5sqxfnkrrd-cups-2.3.3-dev/include/cups/cups.h:27,
system-config-printer>                  from udev/udev-configure-printer.c:36:
system-config-printer> /nix/store/af9nj6nd93m4zq81kg1nxa5sqxfnkrrd-cups-2.3.3-dev/include/cups/http.h:458:17: note: declared here
system-config-printer>   458 | extern http_t  *httpConnectEncrypt(const char *host, int port, http_encryption_t encryption) _CUPS_DEPRECATED_1_7_MSG("Use httpConnect2 instead.");
system-config-printer>       |                 ^~~~~~~~~~~~~~~~~~
system-config-printer> gcc  -I/nix/store/8h40slssggzpvb5lr7irp8xa42qajv88-libusb-1.0.24-dev/include/libusb-1.0 -I/nix/store/6l8vy0rpikcw0j1xw67qf983sqpxcfzp-glib-2.68.3-dev/include/glib-2.0 -I/nix/store/i6z36ml0nm0vl93q9jn40bwjbi5b6r8g-glib-2.68.3/lib/glib-2.0/include -g -O2   -o udev/udev-configure-printer udev/udev_configure_printer-udev-configure-printer.o -lcups -ludev -L/nix/store/sr7wxpq41mxxi5cmaqz3ys5475xi9lpl-libusb-1.0.24/lib -lusb-1.0 -L/nix/store/i6z36ml0nm0vl93q9jn40bwjbi5b6r8g-glib-2.68.3/lib -lglib-2.0
system-config-printer> xmlto man -o man man/system-config-printer.xml
system-config-printer> Warn: AUTHOR sect.: no personblurb|contrib for Tim Waugh
system-config-printer> Note: AUTHOR sect.: see http://www.docbook.org/tdg5/en/html/contr
system-config-printer> Note: AUTHOR sect.: see http://www.docbook.org/tdg5/en/html/perso
system-config-printer> Note: Writing system-config-printer.1
system-config-printer> Warn: AUTHOR sect.: no personblurb|contrib for Tim Waugh
system-config-printer> Note: AUTHOR sect.: see http://www.docbook.org/tdg5/en/html/contr
system-config-printer> Note: AUTHOR sect.: see http://www.docbook.org/tdg5/en/html/perso
system-config-printer> Note: Writing system-config-printer-applet.1
system-config-printer> touch .stamp-man-pages-built
system-config-printer> LC_ALL=C /nix/store/cf74dzz61z7jahr8qh4hg6yl3swxl9is-intltool-0.51.0/bin/intltool-merge  -x -u -c ./po/.intltool-merge-cache ./po system-config-printer.appdata.xml.in system-config-printer.appdata.xml
system-config-printer> Generating and caching the translation database
system-config-printer> Merging translations into system-config-printer.appdata.xml.
system-config-printer> CREATED system-config-printer.appdata.xml
system-config-printer> LC_ALL=C /nix/store/cf74dzz61z7jahr8qh4hg6yl3swxl9is-intltool-0.51.0/bin/intltool-merge  -d -u -c ./po/.intltool-merge-cache ./po system-config-printer.desktop.in system-config-printer.desktop
system-config-printer> Found cached translation database
system-config-printer> Merging translations into system-config-printer.desktop.
system-config-printer> LC_ALL=C /nix/store/cf74dzz61z7jahr8qh4hg6yl3swxl9is-intltool-0.51.0/bin/intltool-merge  -d -u -c ./po/.intltool-merge-cache ./po print-applet.desktop.in print-applet.desktop
system-config-printer> Found cached translation database
system-config-printer> Merging translations into print-applet.desktop.
system-config-printer> sed \
system-config-printer>   -e "s|\@prefix\@|/nix/store/76v2wq6hlyi4za9jhhj4sf9m630f76g4-system-config-printer-1.5.12|" \
system-config-printer>   -e "s|\@datadir\@|/nix/store/76v2wq6hlyi4za9jhhj4sf9m630f76g4-system-config-printer-1.5.12/share|" \
system-config-printer>   -e "s|\@localedir\@|/nix/store/76v2wq6hlyi4za9jhhj4sf9m630f76g4-system-config-printer-1.5.12/share/locale|" \
system-config-printer>   -e "s|\@VERSION\@|1.5.12|" \
system-config-printer>   -e "s|\@PACKAGE\@|system-config-printer|" \
system-config-printer>   config.py.in > config.py
system-config-printer> sed \
system-config-printer> -e "s|\@bindir\@|/nix/store/76v2wq6hlyi4za9jhhj4sf9m630f76g4-system-config-printer-1.5.12/bin|" \
system-config-printer> dbus/org.fedoraproject.Config.Printing.service.in >dbus/org.fedoraproject.Config.Printing.service
system-config-printer> if [ "." != "." ]; then  \
system-config-printer>  cp ./setup.py .;                        \
system-config-printer>  /nix/store/nvlnl2swxmk50x25mzn71rhc0gzpylfr-coreutils-8.32/bin/mkdir -p cupshelpers;                            \
system-config-printer>  for file in cupshelpers/__init__.py cupshelpers/cupshelpers.py cupshelpers/installdriver.py cupshelpers/ppds.py cupshelpers/openprinting.py cupshelpers/xmldriverprefs.py; do           \
system-config-printer>          cp ./$file $file;               \
system-config-printer>  done;                                           \
system-config-printer> fi
system-config-printer> touch .stamp-distutils-in-builddir
system-config-printer> /nix/store/nvlnl2swxmk50x25mzn71rhc0gzpylfr-coreutils-8.32/bin/mkdir -p cupshelpers
system-config-printer> sed \
system-config-printer>   -e "s|\@prefix\@|/nix/store/76v2wq6hlyi4za9jhhj4sf9m630f76g4-system-config-printer-1.5.12|" \
system-config-printer>   -e "s|\@sysconfdir\@|/nix/store/76v2wq6hlyi4za9jhhj4sf9m630f76g4-system-config-printer-1.5.12/etc|" \
system-config-printer>   -e "s|\@cupsserverbindir\@|/nix/store/a8nvssqmbkd953ibbais2jp8f9vvs85l-cups-2.3.3/lib/cups|" \
system-config-printer>   cupshelpers/config.py.in > cupshelpers/config.py
system-config-printer> /nix/store/yqjhkdarf7p74wcha7aypzsb751n7wl2-python3-3.9.5/bin/python setup.py build
system-config-printer> running build
system-config-printer> running build_py
system-config-printer> creating build
system-config-printer> creating build/lib
system-config-printer> creating build/lib/cupshelpers
system-config-printer> copying cupshelpers/config.py -> build/lib/cupshelpers
system-config-printer> copying cupshelpers/ppds.py -> build/lib/cupshelpers
system-config-printer> copying cupshelpers/openprinting.py -> build/lib/cupshelpers
system-config-printer> copying cupshelpers/cupshelpers.py -> build/lib/cupshelpers
system-config-printer> copying cupshelpers/xmldriverprefs.py -> build/lib/cupshelpers
system-config-printer> copying cupshelpers/installdriver.py -> build/lib/cupshelpers
system-config-printer> copying cupshelpers/__init__.py -> build/lib/cupshelpers
system-config-printer> make[1]: Leaving directory '/build/source'
system-config-printer> running tests
system-config-printer> check flags: SHELL=/nix/store/b45zavallnsvqwjs9wg9xw167jcs0935-bash-4.4-p23/bin/bash VERBOSE=y check
system-config-printer> Making check in po
system-config-printer> make[1]: Entering directory '/build/source/po'
system-config-printer> rm -f missing notexist
system-config-printer> srcdir=. /nix/store/cf74dzz61z7jahr8qh4hg6yl3swxl9is-intltool-0.51.0/bin/intltool-update -m
system-config-printer> if [ -r missing -o -r notexist ]; then \
system-config-printer>   exit 1; \
system-config-printer> fi
system-config-printer> make[1]: Leaving directory '/build/source/po'
system-config-printer> make[1]: Entering directory '/build/source'
system-config-printer> /nix/store/yqjhkdarf7p74wcha7aypzsb751n7wl2-python3-3.9.5/bin/python setup.py build
system-config-printer> running build
system-config-printer> running build_py
system-config-printer> make  check-TESTS
system-config-printer> make[2]: Entering directory '/build/source'
system-config-printer> make[3]: Entering directory '/build/source'
system-config-printer> echo "#!/nix/store/b45zavallnsvqwjs9wg9xw167jcs0935-bash-4.4-p23/bin/bash" > "test-xml-validity.sh"
system-config-printer> echo "set -e" >> "test-xml-validity.sh"
system-config-printer> echo "xmllint --relaxng \\" >> "test-xml-validity.sh"
system-config-printer> echo "    ./xml/preferreddrivers.rng \\" >> "test-xml-validity.sh"
system-config-printer> echo "    ./xml/preferreddrivers.xml >/dev/null" >> "test-xml-validity.sh"
system-config-printer> echo "/nix/store/yqjhkdarf7p74wcha7aypzsb751n7wl2-python3-3.9.5/bin/python ./xml/validate.py \\" >> "test-xml-validity.sh"
system-config-printer> echo "    ./xml/preferreddrivers.xml" >> "test-xml-validity.sh"
system-config-printer> chmod 755 "test-xml-validity.sh"
system-config-printer> FAIL: test-xml-validity.sh
system-config-printer> echo "#!/nix/store/b45zavallnsvqwjs9wg9xw167jcs0935-bash-4.4-p23/bin/bash" > "py.test.sh"
system-config-printer> echo "exec " >> "py.test.sh"
system-config-printer> chmod 755 "py.test.sh"
system-config-printer> PASS: py.test.sh
system-config-printer> ====================================================
system-config-printer>    system-config-printer 1.5.12: ./test-suite.log
system-config-printer> ====================================================
system-config-printer> # TOTAL: 2
system-config-printer> # PASS:  1
system-config-printer> # SKIP:  0
system-config-printer> # XFAIL: 0
system-config-printer> # FAIL:  1
system-config-printer> # XPASS: 0
system-config-printer> # ERROR: 0
system-config-printer> .. contents:: :depth: 2
system-config-printer> FAIL: test-xml-validity.sh
system-config-printer> ==========================
system-config-printer> ./xml/preferreddrivers.xml validates
system-config-printer> Validating ./xml/preferreddrivers.xml
system-config-printer> Traceback (most recent call last):
system-config-printer>   File "/build/source/./xml/validate.py", line 79, in <module>
system-config-printer>     if not validator.validate ():
system-config-printer>   File "/build/source/./xml/validate.py", line 38, in validate
system-config-printer>     (drivertypes, preferenceorder) = preferreddrivers.getchildren ()
system-config-printer> AttributeError: 'xml.etree.ElementTree.Element' object has no attribute 'getchildren'
system-config-printer> FAIL test-xml-validity.sh (exit status: 1)
system-config-printer> ============================================================================
system-config-printer> Testsuite summary for system-config-printer 1.5.12
system-config-printer> ============================================================================
system-config-printer> # TOTAL: 2
system-config-printer> # PASS:  1
system-config-printer> # SKIP:  0
system-config-printer> # XFAIL: 0
system-config-printer> # FAIL:  1
system-config-printer> # XPASS: 0
system-config-printer> # ERROR: 0
system-config-printer> ============================================================================
system-config-printer> See ./test-suite.log
system-config-printer> ============================================================================
system-config-printer> make[3]: *** [Makefile:1416: test-suite.log] Error 1
system-config-printer> make[3]: Leaving directory '/build/source'
system-config-printer> make[2]: *** [Makefile:1524: check-TESTS] Error 2
system-config-printer> make[2]: Leaving directory '/build/source'
system-config-printer> make[1]: *** [Makefile:1770: check-am] Error 2
system-config-printer> make[1]: Leaving directory '/build/source'
system-config-printer> make: *** [Makefile:1301: check-recursive] Error 1
error: builder for '/nix/store/cfj5q7lizvimbp1ricjdzdfrbmvpgji2-system-config-printer-1.5.12.drv' failed with exit code 2;
       last 10 log lines:
       > ============================================================================
       > See ./test-suite.log
       > ============================================================================
       > make[3]: *** [Makefile:1416: test-suite.log] Error 1
       > make[3]: Leaving directory '/build/source'
       > make[2]: *** [Makefile:1524: check-TESTS] Error 2
       > make[2]: Leaving directory '/build/source'
       > make[1]: *** [Makefile:1770: check-am] Error 2
       > make[1]: Leaving directory '/build/source'
       > make: *** [Makefile:1301: check-recursive] Error 1
       For full logs, run 'nix log /nix/store/cfj5q7lizvimbp1ricjdzdfrbmvpgji2-system-config-printer-1.5.12.drv'.
```
</details>

**Notify maintainers**
<!--
Please @ people who are in the `meta.maintainers` list of the offending package or module.
If in doubt, check `git blame` for whoever last touched something.
-->
There aren't any maintainers listed :(

**Metadata**
 - system: `"x86_64-linux"`
 - host os: `Linux 5.12.14-zen1, NixOS, 21.11.20210717.b59c06d (Porcupine)`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.4pre20210601_5985b8b`
 - nixpkgs: `/nix/store/r132byyc68d515ss7ma6gai7zffv56vr-source`

Maintainer information:
```yaml
# a list of nixpkgs attributes affected by the problem
attribute: system-config-printer
# a list of nixos modules affected by the problem
module:
```

--- a/pkgs/tools/misc/system-config-printer/default.nix
+++ b/pkgs/tools/misc/system-config-printer/default.nix
@@ -8,13 +8,13 @@
 
 stdenv.mkDerivation rec {
   pname = "system-config-printer";
-  version = "1.5.12";
+  version = "1.5.15";
 
   src = fetchFromGitHub {
     owner = "openPrinting";
     repo = pname;
-    rev = version;
-    sha256 = "1a812jsd9pb02jbz9bq16qj5j6k2kw44g7s1xdqqkg7061rd7mwf";
+    rev = "v${version}";
+    sha256 = "0a3v8fp1dfb5cwwpadc3f6mv608b5yrrqd8ddkmnrngizqwlswsc";
   };
 
   prePatch = ''
@@ -26,17 +26,6 @@ stdenv.mkDerivation rec {
 
   patches = [
     ./detect_serverbindir.patch
-
-    # https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=958104
-    # (Fixes will be included in next upstream release.)
-    (fetchpatch {
-      url = "https://github.com/OpenPrinting/system-config-printer/commit/cf9903466c1a2d18a701f3b5e8c7e03483e1244d.patch";
-      sha256 = "03gpav618w50q90m2kdkgwclc7fv17m493fgjd633zfavb5kqr3n";
-    })
-    (fetchpatch {
-      url = "https://github.com/OpenPrinting/system-config-printer/commit/b9289dfe105bdb502f183f0afe7a115ecae5f2af.patch";
-      sha256 = "12w47hy3ly4phh8jcqxvdnd5sgbnbp8dnscjd7d5y2i43kxj7b23";
-    })
   ];
 
   buildInputs = [
@@ -54,6 +43,10 @@ stdenv.mkDerivation rec {
   ];
 
   pythonPath = with python3Packages; requiredPythonModules [ pycups pycurl dbus-python pygobject3 requests pycairo pysmbc ];
+
+  preConfigure = ''
+    intltoolize --copy --force --automake
+  '';
 
   configureFlags = [
     "--with-udev-rules"

